examples/python/data/copy_stencil_ref.cpp

```diff
@@ -0,0 +1,168 @@
+namespace cuda{
```

Comment:
is there a reason why we want the reference to be generated files rather than serialized IIR-files?

Reply:
just for other users to be able to read this. But no strong reason

Reply:
sounds like a good reason to me